lib/utils/index.js

```diff
@@ -579,6 +581,55 @@ class Utils {
       throw err;
     }
   }
+
+  static iosRealDeviceUDID(udid) {
+    if (udid.length > 25) {
+      return udid;
+    }  
+    
+    if (udid.length < 24) {
+      throw new Error('Incorrect UDID provided for real iOS device');
+    }
+
+    return `${udid.substring(0, 8)}-${udid.substring(9, 25)}`;
+  };
+
+  static getSdkRootFromEnv() {
+    const androidHomeInGlobalEnv = 'ANDROID_HOME' in process.env;
+
+    dotenv.config({path: path.join(__dirname, '.env')});
```

Comment:
No need for the above two lines. `dotenv` is already enabled in Nightwatch. (But please check if it is enabled from this point).